tts/Object.def.lua

```diff
@@ -494,3 +491,13 @@ end
 ---@return tts__Object[]
 function getAllObjects()
 end
+
+--- Render vector lines from a list of parameters, relative the aassoicated objects
+---@param parameters tts__VectorLineGroup[]
+function Object.setVectorLines(parameters) end
+
+--- Returns a table of sub-tables, each sub-table representing VectorLineGroup.
+---@return tts__VectorLineGroup[]
+function Object.getVectorLines()
+    return {}
```

Comment:
If you could please omit the return statements that'd be appreciated. 

Instead you'll want to disable the return inspection for `.def.lua` files:
https://github.com/Benjamin-Dobell/IntelliJ-Luanalysis/issues/7

Reply:
Fixed, but I will tell you that document is currently incorrect.I had to use the pattern `file:*.def.lua` without the `**/`

Reply:
May be because you're using source roots where as I'm using a sub-directory. If you want to add a comment about that on the issue so that others may benefit, that'd be appreciated.